sheetstorm/management/commands/generate_test_data.py

```diff
@@ -61,6 +75,42 @@ def _set_user_factory_parameters(user_type):
             "is_superuser": user_type == superuser_user_type,
         }
 
+    @staticmethod
+    def _get_project_options(options):
+        return {
+            suspended_project: options[suspended_project] - Project.objects.filter_suspended().count()
+            if options[suspended_project] is not None
+            else 0,
```

Comment:
You can extract this part to separate method with parameters as switcher for suspended/active/completed projects.

Reply:
Done